app/views/reports/regions/index.html.erb

```diff
@@ -2,50 +2,29 @@
   <div class="row">
     <div class="col-sm-5 col-lg-7">
 
-    <% if region_reports_enabled? %>
       <%= form_tag regions_search_url(format: :json), id: "search-form", method: :get do %>
         <div class="mo-search">
           <%= render "search" %>
         <div>
       <% end %>
 
-
-       <div id="facility-access">
-          <% @accessible_regions.each do |org, children| %>
-            <div class="card show region-index">
-              <div class="access-tree__section-header">
-                <%= org.name %>
-              </div>
-
-              <%= render Reports::RegionTreeComponent.new(parent: org, children: children) %>
+      <div id="facility-access">
+        <% @accessible_regions.each do |org, children| %>
+          <div class="card organization show region-index">
+            <div class="access-tree__section-header">
+              <%= org.name %>
```

Comment:
What will this look like in Bangladesh?

Reply:
It should be the same, because we restrict to reportable children, which only returns district and facility there.